external/@wordpress/elements/index.js

```diff
@@ -10,7 +16,7 @@
  * @return {wp.Element}                  Element
  */
 export function createElement( type, props, ...children ) {
```

Comment:
Well! the current approach is more "verbose" but it's self-documenting the fact that it's an abstraction.